yaml.go

```diff
@@ -278,6 +278,11 @@ func (e *Encoder) SetIndent(spaces int) {
 	e.encoder.indent = spaces
 }
 
+// DisableSeqIndent disables the indentation for sequence items.
+func (e *Encoder) DisableSeqIndent() {
+	e.encoder.emitter.disable_sequence_indent = true
```

Comment:
is there a way this can be called prior to the encoder and emitter being instantiated?

Reply:
good question, though I'm wondering why we would want to do that?